pkg/oomstore/cache.go

```diff
@@ -14,12 +14,12 @@ import (
 	"github.com/oom-ai/oomstore/pkg/oomstore/types"
 )
 
-const (
-	Day       = 24 * time.Hour
-	Capacity  = 1000
-	Period    = 5 * time.Minute
-	MinPeriod = 2 * time.Minute
-)
+var defaultCfg = types.StreamPushProcessorConfig{
+	RevisionInterval: 24 * time.Hour,
+	Period:           5 * time.Minute,
+	MinPeriod:        2 * time.Minute,
```

Comment:
I don't think we need both `Period` and `MinPeriod`. Just one `FlushInterval` is enough?

Reply:
if I remember correctly, `MinPeriod` is to avoid too frequent push

Reply:
I think this is open to discussion, but it is not appropriate to change in this PR.

Reply:
@lianxmfor Ok, I can do this refactor later.